src/models/keyframesmodel.h

```diff
@@ -86,6 +86,8 @@ public slots:
     void reload();
     void onFilterChanged(const QString& property);
     void onFilterInChanged(int delta);
+    void trimFilterIn(int in);
```

Comment:
Not sure about the location of these functions. The QML needs a function to call that is not the filter so that the adjustments can be made. Could also go in keyframesdock, for example.

Reply:
This is fine. It is a little arbitrary, but conceptually it can be argued these methods are model-oriented, and there is a `onFilterInChanged` slot here making it sensible to group them together.